samples/Azure.ResourceManager.Sample/Generated/Models/AvailabilitySetData.cs

```diff
@@ -26,15 +26,15 @@ public AvailabilitySetData(LocationData location) : base(location)
         /// <param name="id"> The id. </param>
         /// <param name="name"> The name. </param>
         /// <param name="type"> The type. </param>
-        /// <param name="tags"> The tags. </param>
         /// <param name="location"> The location. </param>
+        /// <param name="tags"> The tags. </param>
         /// <param name="sku"> Sku of the availability set, only name is required to be set. See AvailabilitySetSkuTypes for possible set of values. Use &apos;Aligned&apos; for virtual machines with managed disks and &apos;Classic&apos; for virtual machines with unmanaged disks. Default value is &apos;Classic&apos;. </param>
         /// <param name="platformUpdateDomainCount"> Update Domain count. </param>
         /// <param name="platformFaultDomainCount"> Fault Domain count. </param>
         /// <param name="virtualMachines"> A list of references to all virtual machines in the availability set. </param>
         /// <param name="proximityPlacementGroup"> Specifies information about the proximity placement group that the availability set should be assigned to. &lt;br&gt;&lt;br&gt;Minimum api-version: 2018-04-01. </param>
         /// <param name="statuses"> The resource status information. </param>
-        internal AvailabilitySetData(ResourceGroupResourceIdentifier id, string name, ResourceType type, IDictionary<string, string> tags, LocationData location, Sku sku, int? platformUpdateDomainCount, int? platformFaultDomainCount, IList<SubResource> virtualMachines, SubResource proximityPlacementGroup, IReadOnlyList<InstanceViewStatus> statuses) : base(id, name, type, tags, location)
+        internal AvailabilitySetData(ResourceGroupResourceIdentifier id, string name, ResourceType type, LocationData location, IDictionary<string, string> tags, Sku sku, int? platformUpdateDomainCount, int? platformFaultDomainCount, IList<SubResource> virtualMachines, SubResource proximityPlacementGroup, IReadOnlyList<InstanceViewStatus> statuses) : base(id, name, type, location, tags)
```

Comment:
This parameter sequencing is biting us today. We should consider that adding full (parameter: parameter) in our caller code (see VMimageContainer.cs). Can you add an ADO to track it?

Reply:
yeah, this was ordering it according to the generated swagger vs before it was hand written in the wrong order, so all of these got updated to be correct now